docs/docs-content/clusters/edge/trusted-boot/deployment-day2/install.md

```diff
@@ -0,0 +1,164 @@
+against the installer ISO and provider image. With secure boot enabled and keys enrolled, you won't be able to boot from
+the EFI size checking ISO unless you disable secure boot or clear the keys. For more information, refer to
+[Check EFI Size and Edge Host Boot Limit](../edgeforge/check-efi-limit.md).
+
+:::
+
+## Instructions
+
+### Enroll Secure Boot Keys into Edge Device
+
+The following steps are written for the Intel NUC 13 Pro device. If you are using a different device, the exact sequence
+of steps might be different. However, you should be able to find the same or similarly worded options in any device that
+supports Secure Boot.
+
+1. Insert the USB disk into the Edge device.
+
+2. Power up your Edge device and enter the BIOS interface. Oftentimes you can accomplish this by pressing F2, F1, or F10
+   on the keyboard immediately after powering up the deice and before you select a boot volume, as is the case in Intel
+   NUC 13 Pro. However, the exact method might be different on your Edge device. Consult the manufacturer of your Edge
+   device to find out how to enter the BIOS interface.
+
+3. From the BIOS menu, select the **Boot** tab. Then select **Secure Boot**.
+
+4. From the **Secure Boot Menu**, make sure that secure boot is enabled. Then select **Restore to Setup Mode**. Two
+   dialogue boxes will pop up to confirm this operation. Select **Yes** to confirm. This will make the device restart.
+
+   :::info
+
+   **User Mode** and **Setup Mode** are UEFI terms. Setting secure boot mode to **Setup Mode** means clearing the keys
+   that are currently in the device. This allows you to enroll your own keys, while **User Mode** means you have
+   enrolled your own keys.
+
+   In some BIOS, the same actions or functionality might be named differently in the BIOS interface. Consult the
+   manufacturer of your Edge host to find out how to clear the keys and enroll your own keys.
+
+   :::
+
+If you set the `AUTO_ENROLL_SECUREBOOT_KEYS` argument to `true` when you built the installation ISO, key enrollment will
+begin automatically once the device restarts. You can skip the following steps to proceed to
+[Install Palette Edge with Trusted Boot](#install-palette-edge-with-trusted-boot).
+
+5. If you did not set `AUTO_ENROLL_SECUREBOOT_KEYS` to `true` when you built the ISO, you will need to select the
+   **Enroll Secure Boot keys: Auto** option in the boot menu when the Edge device restarts. This will start the key
+   enrollment process.
+
+### Install Palette Edge with Trusted Boot
+
+6. When the keys are finished enrolling, the installation will start automatically after a reboot. Installation
+   typically takes about 5 minutes, but can vary depending on your hardware.
+
+7. When installation finishes, you will observe the text "Installation has finished, rebooting in 5 seconds" on your
+   screen. Remove the USB disk from your Edge device.
```

Comment:
at this point, from a security perspective, it is also advisable to ensure the boot order places the target installation drive first.

Reply:
This would require that the users go into the BIOS menu again, right? Can you elaborate a bit why this is necessary?

Reply:
because you don't want the box to ever boot from anything except the main system drive post installation. attackers will try to oinsert usb disks and see if you did for some reason left usb as the first boot option. 
this is an important part of preparing your hardware for production. all other boot options should be disabled or removed from teh boot order once installation is completed.

Reply:
for a better and more secure flow, what I do is this:
- I leave the system drive as the first boot option
- I manually in the bios tell it to boot off the usb stick during enrollment/installation (yes, it means I need to do that a couple of times)
- I look to see if I can remove any other potential boot options such as network boot, usb, etc.

We don't need to go into details here, but we should give the high level message to people that after installation, they should ensure that their device boot options are set securely and only the system drive should be bootable.

Reply:
Note - this does not say that trusted boot wouldn't prevent booting a rogue usb stick, this is called defence in depth - just because we have trusted boot doesn't mean we (or our customers) shouldn't take all normal security precautions during device setup.